ui/src/components/DataTable/index.jsx

```diff
@@ -76,49 +99,83 @@ export default class DataTable extends Component {
     }
   };
 
+  handleChangePage = (event, page) => {
+    this.setState({ page });
+  };
+
   render() {
     const {
+      classes,
       items,
       columnsSize,
       renderRow,
       headers,
       sortByHeader,
       sortDirection,
       noItemsMessage,
+      isPaginate,
     } = this.props;
     const colSpan = columnsSize || (headers && headers.length) || 0;
+    const { page } = this.state;
+    const rowsPerPage = 5;
 
     return (
-      <Table>
-        {headers && (
-          <TableHead>
-            <TableRow>
-              {headers.map(header => (
-                <TableCell key={`table-header-${header}`}>
-                  <TableSortLabel
-                    id={header}
-                    active={header === sortByHeader}
-                    direction={sortDirection || 'desc'}
-                    onClick={this.handleHeaderClick}>
-                    {header}
-                  </TableSortLabel>
-                </TableCell>
-              ))}
-            </TableRow>
-          </TableHead>
+      <Fragment>
+        <div className={classes.tableWrapper}>
+          <Table className={classes.table}>
+            {headers && (
+              <TableHead>
+                <TableRow>
+                  {headers.map(header => (
+                    <TableCell key={`table-header-${header}`}>
+                      <TableSortLabel
+                        id={header}
+                        active={header === sortByHeader}
+                        direction={sortDirection || 'desc'}
+                        onClick={this.handleHeaderClick}>
+                        {header}
+                      </TableSortLabel>
+                    </TableCell>
+                  ))}
+                </TableRow>
+              </TableHead>
+            )}
+            <TableBody>
+              {items.length === 0 ? (
+                <TableRow>
+                  <TableCell colSpan={colSpan}>
+                    <em>{noItemsMessage}</em>
+                  </TableCell>
+                </TableRow>
+              ) : (
+                items
+                  .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+                  .map(renderRow)
+              )}
+            </TableBody>
+          </Table>
+        </div>
+        {isPaginate && (
+          <TablePagination
+            classes={{
+              spacer: classes.noDisplay,
+              caption: classes.noDisplay,
+            }}
+            component="div"
+            count={items.length}
+            rowsPerPage={rowsPerPage}
+            rowsPerPageOptions={[rowsPerPage]}
+            page={page}
+            backIconButtonProps={{
+              'aria-label': 'Previous Page',
+            }}
+            nextIconButtonProps={{
+              'aria-label': 'Next Page',
+            }}
```

Comment:
Nice, I like the `backIconButtonProps` and `nextIconButtonProps`. Do you mind adding it to `ConnectionDataTable` as well, please :)